apps/create-vesting/src/app/page.tsx

```diff
@@ -1,126 +1,162 @@
-                className="h-12 p-2.5 rounded-xl bg-color-paper-darker border-none placeholder:opacity-100"
-              />
-              <br />
-              <div className="flex gap-4 w-full">
-                <PeriodWithScaleInput
-                  periodScaleOptions={periodScaleOptions}
-                  namePeriodValue="period"
-                  namePeriodScale="periodScale"
-                  label="Period"
-                  validation={{ valueAsNumber: true, required: true }}
-                  onKeyDown={(e) =>
-                    ["e", "E", "+", "-"].includes(e.key) && e.preventDefault()
-                  }
-                />
-                <TokenAmountInput
-                  name="amount"
-                  tokenSymbol="ETH"
-                  label="Amount"
-                  placeholder="0.0"
-                  autoComplete="off"
-                  validation={{ valueAsNumber: true, required: true }}
-                  onKeyDown={(e) =>
-                    ["e", "E", "+", "-"].includes(e.key) && e.preventDefault()
-                  }
-                />
-              </div>
-              <br />
-            </ContentWrapper>
-            <ButtonPrimary type="submit" onClick={handleSubmit(onSubmit)}>
-              <span>{context?.hookToEdit ? "Edit Hook" : "Add hook"}</span>
-            </ButtonPrimary>
-          </Wrapper>
-        </Form>
-      )}
-    </>
+    <Form {...form} onSubmit={onSubmit} className="contents">
+      <Wrapper>
+        <ContentWrapper>
+          <Input
+            name="recipient"
+            label="Recipient"
+            placeholder="0xabc..."
+            autoComplete="off"
+            className="h-12 p-2.5 rounded-xl bg-color-paper-darker border-none placeholder:opacity-100"
+          />
+          <br />
+          <div className="flex flex-col w-full xsm:gap-4 xsm:flex-row">
+            <PeriodWithScaleInput
+              periodScaleOptions={periodScaleOptions}
+              namePeriodValue="period"
+              namePeriodScale="periodScale"
+              type="number"
+              step="0.0000001"
+              label="Period"
+              validation={{ valueAsNumber: true, required: true }}
+              onKeyDown={(e) =>
+                ["e", "E", "+", "-"].includes(e.key) && e.preventDefault()
+              }
+            />
+            <br className="xsm:h-0 xsm:w-0" />
+            <TokenAmountInput
+              name="amount"
+              type="number"
+              step={`0.${"0".repeat(tokenDecimals ? tokenDecimals - 1 : 8)}1`}
+              token={token}
+              label="Amount"
+              placeholder="0.0"
+              autoComplete="off"
+              disabled={vestAllFromSwap}
+              validation={{ valueAsNumber: true, required: true }}
+              onKeyDown={(e) =>
+                ["e", "E", "+", "-"].includes(e.key) && e.preventDefault()
+              }
+            />
+          </div>
+          <br />
```

Comment:
I would prefer to use flex components and the gap, margin properties than using <br/>

Reply:
I put the div more because of mobile, if mobile flex-col else flex-row, but this problem will no longer exist in the new bibi UI